src/app_layout.test.js

```diff
@@ -0,0 +1,42 @@
+import { getEnvUrl, initWithWalletConnected } from '../utils/testSetup'
+import { appLayout } from '../utils/selectors/appLayout'
+import config from '../utils/config'
+
+let browser
+let gnosisPage
+let envUrl
+
+const { TESTING_SAFE_ADDRESS } = config
+
+beforeAll(async () => {
+  ;[browser, , gnosisPage] = await initWithWalletConnected()
+  envUrl = getEnvUrl()
+}, 60000)
+
+afterAll(async () => {
+  await gnosisPage.waitForTimeout(5000)
+  const pages = await browser.pages()
+  await Promise.all(pages.map((page) => page.close()))
+  await browser.close()
+})
+
+describe('Navigate through the app routes', () => {
```

Comment:
I think this description is not super accurate

Reply:
I agree it could be more specific, any suggestion?

Reply:
Usually `describe` block contains a description about what the test suite will test. For this one, I'd just make it `describe('Application Layout')` 😄